plush_test.go

```diff
@@ -342,7 +342,32 @@ func Test_UndefinedArg(t *testing.T) {
 
 	_, err := Render(input, ctx)
 	r.Error(err)
-	r.Equal(ErrUnknownIdentifier, errors.Cause(err))
```

Comment:
why did you remove this line in the test? does this PR change that unknown identifier functionality?

Reply:
That's a typo... Thanks!